test/acceptance/helm-chart.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/zsh
+
+set -eux
+set -o pipefail
+
+LATEST_COMMIT_HASH=$(git rev-parse HEAD | cut -c 1-8)
+RELEASE="acceptance-kai-$LATEST_COMMIT_HASH"
+function install_helm () {
+  curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+  chmod 700 get_helm.sh
+  ./get_helm.sh
+}
+
+function install_kubectl () {
+  curl -L -o /usr/local/bin/kubectl "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+  chmod 700 /usr/local/bin/kubectl
+}
+
+function cleanup () {
+  helm uninstall "$RELEASE"
+}
+trap cleanup EXIT
+
+helm version || install_helm
+kubectl version || install_kubectl
+
+helm repo add anchore https://charts.anchore.io
+
+helm install "$RELEASE" -f ./test/acceptance/fixtures/helm/values.yaml anchore/kai
+
+max_iterations=20
+iterations=0
+while [[ $(kubectl get pods -l app.kubernetes.io/name=kai -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]];
+do
+  echo "waiting for pod to be ready" && sleep 1
+  ((iterations++))
+  if [[ "$iterations" -ge "$max_iterations" ]]; then
+    echo "Timeout Waiting for pod"
+    exit 1
+  fi
+done
+
+echo "KAI Helm Chart Successfully Installed, removing"
```

Comment:
nit: I might move the "removing" part to its own `echo` command inside the `cleanup` function, right above the uninstall command, just to keep related things together, esp. since control flow is involved here with the `trap`

Reply:
Fixed it when i was doing [integration tests](https://github.com/anchore/kai/pull/19)! I'll merge that up to here before merging this one cuz it affects acceptance tests too, which might need a few iterations on main to get solid (by nature of acceptance tests being scoped to `main` branch changes)